### PR TITLE
implemented spec

### PR DESCRIPTION
**Pull request by AshiraFriedman**
Please review my work, let me know if there are any corrections

### CODE REVIEW

File: 3 reports.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+where c.Half in ('second', 'both')
+order by c.Grade, c.School
+
+--4) Please calculate the number of girls attending camp for just first half, for just second half, and for both halves.
+select attendance = count(c.CamperId), c.Half
+from Camper c
+group by c.Half
+
+--5) Show me a list of campers with allergies and add the word (warning!) to their last name, ex. Steinman (warning!). 
+select c.FirstName, LastName = concat(c.LastName, ' warning!'), c.Allergies
```
Comment (EstiSchottenstein) [inline 3 reports.sql]: The word warning should be in parenthesis like this: (warning!)

File: 2 data.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+union select 'Bracha', 'Katz', 1, 'Beis Leah', 2163456789, 2165678910, null, 'Gitty Kohn', 'first', 0, 1
+union select 'Gitty', 'Kohn', 1, 'Beis Leah', 2164567890, 2162468102, 'peanuts', 'Bracha Katz', 'first', 1, 0
+union select 'Sarah', 'Baum', 2, 'Beis Rivka', 2166782345, 2169876543, null, 'Rochel Berman', 'both', 1, 0
+union select 'Miriam', 'Cohen', 2, 'Beis Rochel', 2165678910, 2167654321, 'dairy', null, 'second', 0, 0
+union select 'Rivky', 'Cohen', 1, 'Beis Rochel', 2165678910, 2167654321, null, null, 'second', 0, 0
+union select 'Shaindy', 'Bauman', 1, 'Beis Rochel', 2168901234, 7324356759, null, 'Shayna Stein', 'first', 1, 1
+union select 'Rivka', 'Katz', 2, 'Beis Leah', 2163845829, 2165093968, null, null, 'second', 1, 1
+union select 'Avigayil', 'Steinman', 2, 'Beis Sarah', 2164931984, 8482842339, 'dairy', null, 'both', 0, 1
+union select 'Rochel', 'Berman', 2, 'Beis Sarah', 2162081945, 2163058293, null, 'Sarah Baum', 'both', 0, 1
+union select 'Chana', 'Braun', 1,' Beis Sarah', 2169852492, 7322038038, null, null, 'both', 1, 1
```
Comment (EstiSchottenstein) [inline 2 data.sql]: There is an extra space before the name of the school.

**Review by EstiSchottenstein — CHANGES_REQUESTED** [review]
I reviewed your work. Everything looks good, there were just two small mistakes. Please see my inline comments.

File: 2 data.sql (hunk 2 above)
Comment (AshiraFriedman) [reply 2 data.sql]: I removed it

**Pull request closed without merging**